astropy/utils/masked/tests/test_function_helpers.py

```diff
@@ -1453,6 +1453,185 @@ def test_nanpercentile(self):
         self.check(np.nanpercentile, q=50)
 
 
+class TestArraySetOps:
+    """Tests based on those from numpy.ma.tests.test_extras.
+
+    Adjusted to take into account that comparing masked values should
+    result in masked equality.
+
+    """
+
+    @classmethod
+    def setup_class(self):
+        # Setup for unique.
+        self.data = Masked([1, 1, 1, 2, 2, 3], mask=[0, 0, 1, 0, 1, 0])
+        self.values = Masked([1, 2, 3, 1, 2], mask=[0, 0, 0, 1, 1])
```

Comment:
it's not clear to me, by just reading this setup function, what's the difference between those two broadly named attributes. If the names matter, maybe we could use something clearer. If not, `a` and `b` would probably be just fine (and conforming to the local dominant idiom)

Reply:
I expanded the comment - the names are taken from `np.unique` (except for `data`, which is `ar` there...)